src/triangulation.jl

```diff
@@ -22,7 +32,8 @@ function _triangulation(Δs, Δ, v_idx, h_idx, incident_idx, is_weak_adjacent)
             # it will simply end up calling `_triangulation` with an empty `v_idx`.
             weak_adjacent = [hj for hj in h_idx if hj != h && is_weak_adjacent[(h, hj)]]
             active = [point for point in v_idx if point in incident_idx[h]]
-            _triangulation(Δs, Δ, active, weak_adjacent, incident_idx, is_weak_adjacent)
+            _triangulation(Δs, Δ, active, weak_adjacent, incident_idx, is_weak_adjacent,
+                           depth + 1, maxdepth)
```

Comment:
I might be simpler to replace these 2 arguments by a single one `face_dim` which is decremented

Reply:
Good point, done.